build_tools/travis/install.sh

```diff
@@ -43,18 +43,38 @@ make_conda() {
 }
 
 if [[ "$DISTRIB" == "conda" ]]; then
-    TO_INSTALL="python=$PYTHON_VERSION pip pytest pytest-cov \
-                numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION \
-                cython=$CYTHON_VERSION"
+    TO_INSTALL="python=$PYTHON_VERSION pip pytest pytest-cov"
+
+		if [[ "$NUMPY_VERSION" == "latest" ]]; then
+				TO_INSTALL="$TO_INSTALL numpy"
+		else
+				TO_INSTALL="$TO_INSTALL numpy=$NUMPY_VERSION"
```

Comment:
The `*` wildcard does seem to work, doesn't it? If it doesn't try maybe `NUMPY_VERSION="'*'"`, I never remember string escaping in bash...

After which you can remove all this added custom logic here, I think?